odh/overlays/moc/monitoring/servicemonitors/argocd-servicemonitor.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: monitoring.coreos.com/v1
```

Comment:
I think this should be a separate PR.

Reply:
Scratch that.. I didn't get this bit fully. It should stay here. Sorry. :facepalm: